composer.json

```diff
@@ -18,10 +18,13 @@
         "cakephp/cakephp": "^3.5"
     },
     "require-dev": {
-        "dereuromark/cakephp-ide-helper": "@stable",
+        "dereuromark/cakephp-ide-helper": "dev-feature/directive as 0.14",
```

Comment:
Why do you want to merge an *unstable* feature branch into this repository?

Reply:
I dont.
This is only for testing, as you could have seen the last 8 days the PR is open.
I released now, as such this can go back to normal strategy as outlined above. The final branch/PR should contain this.

Once we are clear on how release should happen, I can fix this up.